pkgs/applications/misc/electrum/default.nix

```diff
@@ -68,7 +68,14 @@ python3.pkgs.buildPythonApplication {
     attrs
     bitstring
     cryptography
-    dnspython
```

Comment:
isn't the dependency relaxation enough to fix the build?

``` patch
--- contrib/requirements/requirements.txt
+++ contrib/requirements/requirements.txt
@@ -10,4 +10,4 @@ attrs>=19.2.0
 
 # Note that we also need the dnspython[DNSSEC] extra which pulls in cryptography,
 # but as that is not pure-python it cannot be listed in this file!
-dnspython>=2.0,<2.1
+dnspython>=2.0
```

Reply:
If they did not break the API, it should

Reply:
Reading the releases notes, it seems they keep backwards compatibility, so I included the patch.